src/legacy/core_plugins/vis_default_editor/public/components/controls/components/number_list/number_list.tsx

```diff
@@ -54,31 +52,27 @@ function NumberList({
   range,
   showValidation,
   unitName,
-  validateAscendingOrder = true,
+  validateAscendingOrder = false,
+  disallowDuplicates = false,
   onChange,
   setTouched,
   setValidity,
 }: NumberListProps) {
   const numberRange = useMemo(() => getRange(range), [range]);
   const [models, setModels] = useState(getInitModelList(numberArray));
-  const [ascendingError, setAscendingError] = useState(EMPTY_STRING);
 
   // set up validity for each model
   useEffect(() => {
-    let id: number | undefined;
-    if (validateAscendingOrder) {
-      const { isValidOrder, modelIndex } = validateOrder(numberArray);
-      id = isValidOrder ? undefined : modelIndex;
-      setAscendingError(
-        isValidOrder
-          ? EMPTY_STRING
-          : i18n.translate('visDefaultEditor.controls.numberList.invalidAscOrderErrorMessage', {
-              defaultMessage: 'The values should be in ascending order.',
```

Comment:
How about using either of these instead:

Place values in ascending order.
Values are not in ascending order.